src/azure-cli/azure/cli/command_modules/eventhubs/custom.py

```diff
@@ -621,7 +621,8 @@ def cli_remove_appgroup_policy(client, resource_group_name, namespace_name, appl
     from azure.cli.core import CLIError
 
     appGroup = client.get(resource_group_name, namespace_name, application_group_name)
-
+    logger.warning(
+        'This operation will do removals based on policy name')
```

Comment:
I don't think with this message customers are able to realize there will be breaking change in the future......

It just says policy name will be used for removals

Reply:
If you want to notify that you won't accept `metric-id,rate-limit-threshold` in the future, pls say it explicitly

Reply:
Hey @evelyn-ys , Sorry for the inconvenience.
Does the Description & warning message makes the good sense now?
Any suggestions are always welcome.